src/react-app/index.css

```
@@ -1,29 +1,41 @@
 }
 
 body {
   background-color: var(--black);
   color: var(--green);
-  font-family: "Courier New", Courier, monospace;
-  font-weight: bold;
+  font-family: var(--font-terminal);
+  font-size: 20pt;
+  -webkit-font-smoothing: none;
+  -moz-osx-font-smoothing: grayscale;
+  text-rendering: optimizeSpeed;
```

Stylelint (17.14.0)

[error] Expected "optimizeSpeed" to be "optimizespeed" (value-keyword-case)

(value-keyword-case)